src/patternfly/components/Table/table.scss

```diff
@@ -747,8 +748,7 @@
   .#{$table}__toggle-icon {
     @include pf-v6-mirror-inline-on-rtl;
 
-    transition: transform var(--#{$table}__toggle--c-button__toggle-icon--TransitionDuration) var(--#{$table}__toggle--c-button__toggle-icon--TransitionTimingFunction);
-    transform: rotate(var(--#{$table}__toggle--c-button__toggle-icon--Rotate));
```

Comment:
Looks like this was removed accidentally? Currently all toggles point down whether they're expanded or not.
<img width="1552" alt="Screenshot 2025-03-11 at 10 16 34 PM" src="https://github.com/user-attachments/assets/29b63708-f796-417c-b15b-3ba4a485489d" />